From 23d21359e35536a4ee8f7be0b245ddbfac909bff Mon Sep 17 00:00:00 2001
From: Tomas Mraz <tomas@openssl.org>
Date: Thu, 19 Oct 2023 08:37:47 +0200
Subject: [PATCH 375/381] Fix Coverity 1547856: memset() uses only the lowest
 byte of c

Reviewed-by: Hugo Landau <hlandau@openssl.org>
Reviewed-by: Tom Cosgrove <tom.cosgrove@arm.com>
(Merged from https://github.com/openssl/openssl/pull/22430)

(cherry picked from commit 0bf18140f491024232beca4e139c8feecfe207e9)
---
 test/rsa_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/rsa_test.c b/test/rsa_test.c
index b2c190b2a4..e489aeaa29 100644
--- a/test/rsa_test.c
+++ b/test/rsa_test.c
@@ -484,7 +484,7 @@ static int test_EVP_rsa_legacy_key(void)
     0x85, 0x20, 0x4f, 0x35, 0x02, 0xfa, 0xda, 0x14, 0x77, 0xfa, 0x08, 0x34,
     0x60, 0xc7, 0x93, 0x72, 0xdc, 0xc4, 0x18, 0x70, 0xc1 };
 
-    memset(msgbuf, 0xdeadbeef, 64);
+    memset(msgbuf, 0xef, 64);
 
     ret = (TEST_ptr((p = BN_bin2bn(p_data, sizeof(p_data), NULL)))
            && TEST_ptr((q = BN_bin2bn(q_data, sizeof(q_data), NULL)))
-- 
2.34.1

